C/CoverM/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase
+
+name = "CoverM"
+version = v"0.6.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://github.com/wwood/CoverM",
+        "0056336d4b245064a23e5b40197b29607fb1fe56"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/CoverM/
+cargo build --release
+install -Dvm 0755 "target/${rust_target}/release/coverm${exeext}" "${bindir}/coverm${exeext}"
+install_license LICENCE.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(supported_platforms()) do platform
+    # Windows can't compile the dep htslib-rust for now, and I don't care about
+    # other platforms
+    (Sys.isapple(platform) || Sys.islinux(platform)) &&
+    in(arch(platform), ("x86_64", "aarch64")) &&
+
+    # aarch64 linux errors, see https://github.com/rust-bio/rust-htslib/issues/352
+    # will be fixed in an upcoming release
+    !(arch(platform) == "aarch64" && Sys.islinux(platform))
```

Comment:
Upcoming?  https://github.com/rust-bio/rust-htslib/commit/a21aff289bc03c7549afc7a958084ed57e8c93f2 is in [rust-htslib-v0.40.2](https://github.com/rust-bio/rust-htslib/releases/tag/rust-htslib-v0.40.2) [rust-htslib-v0.40.1](https://github.com/rust-bio/rust-htslib/releases/tag/rust-htslib-v0.40.1) [rust-htslib-v0.40.0](https://github.com/rust-bio/rust-htslib/releases/tag/rust-htslib-v0.40.0) [rust-htslib-v0.39.5](https://github.com/rust-bio/rust-htslib/releases/tag/rust-htslib-v0.39.5) [rust-htslib-v0.39.4](https://github.com/rust-bio/rust-htslib/releases/tag/rust-htslib-v0.39.4) [rust-htslib-v0.39.3](https://github.com/rust-bio/rust-htslib/releases/tag/rust-htslib-v0.39.3)

Reply:
Ah, but this is using v0.36.0: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=23855&view=logs&j=8bc2bf0b-2b12-5cfb-16e6-a407f47132ec&t=334cd4bd-c913-5a35-b576-74a0aea7bbcf&l=491.  Sigh.

Reply:
Yeah - the `main` branch of CoverM already bumped `rust-htslib`, but I'd rather not use CoverM straight from main. Hopefully when they make a new release I can test on these other platforms and make a new CoverM_jll release.